tests/fixtures/common_test.py

```diff
@@ -4,15 +4,15 @@
 
 
 @pytest.mark.parametrize(
-    'build_only, device_testing, runnable, platform_type, expected',
+    'build_only, device_testing, runnable, platform_type, platform_sim, expected',
     [
-        (True, False, False, 'any', False),
-        (False, True, False, 'any', False),
-        (False, True, False, 'mcu', False),
-        (False, False, True, 'mcu', False),
-        (False, False, True, 'any', True),
+        (True, False, False, 'any', 'na', False),
+        (False, True, False, 'any', 'na', False),
+        (False, True, False, 'mcu', 'na', False),
+        (False, False, True, 'mcu', 'na', False),
+        (False, False, True, 'any', 'na', True),
```

Comment:
You can add a case with platform_sim='qemu'

Reply:
Done